rtl/README.md

```diff
@@ -20,18 +20,84 @@ yarn add @pxblue/storybook-rtl-addon
 In your `main.js` file, register this addon:
 
 ```
-addons: [
-    '@pxblue/storybook-rtl-addon/register',
-]
+addons: ['@pxblue/storybook-rtl-addon/register']
 ```            
 
-> **Angular Components**: If your component uses the Direction service, use the exported function `getDirection()` to supply the `[dir]` directive with the appropriate direction. 
+## Usage
+
+This package exports two functions `getDirection` & `useDirection` that returns the current `Direction` ('rtl' or 'ltr').
+
+### getDirection
+`getDirection` is used within Stories to access the current `Direction`.
+
+For detailed usage for the `getDirection` function, see the Examples section further below.
+
+> **Angular Components**: If your component uses the Direction service, use the exported function `getDirection()` to supply the `[dir]` directive with the appropriate direction.
+
+### useDirection
+`useDirection` is a React hook that emits any `Direction` change and can be used inside of Storybook Decorators.
+
+
+```ts
+import useDirection from "@pxblue/storybook-rtl-addon/useDirection";
+
+export const decorators = [
+    (Story) => {
+        const direction = useDirection();
+        return (
+            <>
+                {direction}
+                <Story />
+            </>
+        );
+    },
+];
+```
+
+## Examples
+
+PX Blue uses this addon extensively in our storybook documentation to guarantee bidirectional support of our components and examples.
+
+To see live-example usage of this addon, click on the "Story" tab of each linked example.
+
+### Angular
+
+```ts
+import { getDirection } from '@pxblue/storybook-rtl-addon';
+
+export const angularExampleStory = () => ({
+    template: `
+        <div>{{direction}}</div>
+    `,
+    props: {
+        direction: getDirection
+    }
+});
+
+```
+
+[Live Example](https://pxblue-components.github.io/angular/?path=/story/components-score-card--with-full-config)
+
+
+### React
+
+```ts
+import { getDirection } from '@pxblue/storybook-rtl-addon';
+
+export const reactExampleStory = () => {
+    const direction = getDirection();
+    return <div>{direction}</div>;
+}
```

Comment:
`getDirection()` in React only gets set on the initial mount, but not update when toggling the storybook sidebar button (the reason for creating the `useDirection` hook - to force a life-cycle update).

Might be worth to note this?